Resources/Prototypes/_StarLight/Catalog/Fills/Items/belt.yml

```diff
@@ -22,4 +22,5 @@
       - id: Handcuffs    
       - id: Brutepack
       - id: Ointment
-      - id: Gauze
\ No newline at end of file
+      - id: Gauze
+      - id: SecurityMedkit
```

Comment:
```suggestion
```

Reply:
You don't want it to be able to fit in the belt?

Reply:
> You don't want it to be able to fit in the belt?

yes